ods_ci/tests/Resources/Files/triton/triton_keras_gRPC_servingruntime.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: serving.kserve.io/v1alpha1
```

Comment:
same as https://github.com/red-hat-data-services/ods-ci/pull/1844/files
Can we avoid duplication and use one file?

Reply:
Then you could use the same filename everytime, so once a PR gets merged you'd just need a rebase. Instead as it is now, you'll need to change all the references to the files in the tests